downstream/modules/platform/ref-ports-firewalls-requirements.adoc

```diff
@@ -1,11 +1,11 @@
 
-[id="ref-access-control-requirements_{context}"]
+[id="ref-ports-firewalls-requirements_{context}"]
 
-= Access control requirements
+= Ports and firewalls requirements
 
 [role="_abstract"]
 
-{PlatformName} uses a number of ports to communicate with its services. These ports must be open and available for incoming connection to the {PlatformName} server in order for it to work. Ensure that these ports are available and are not being blocked by a firewall.
+{PlatformName} (AAP) uses a number of ports to communicate with its services. These ports must be open and available for incoming connection to the {PlatformName} server in order for it to work. Ensure that these ports are available and are not being blocked by the server firewall.
```

Comment:
```suggestion
{PlatformName} (AAP) uses a number of ports to communicate with its services. You must ensure that these ports are open and available for incoming connections to {PlatformName}.
```

Reply:
Suggest compacting these sentences for minimalism. "in order for it to work" is superfluous imo.

Also when you say "server firewall" does that mean that AAP has it's own set of firewall rules? I would think "network firewall" might be a little more accurate here given that it's likely something that customers already have set up in their environment.

Reply:
@oraNod Thanks for this review. I used `server firewall` because the SME specifically asked for those words to be added so that the user knows what to look out for. Because of that, I will leave those two words and use your suggestions for minimalism.